Dockerfile

```
@@ -11,7 +11,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-FROM golang:1.17.3
+FROM golang:1.17.6
```

```suggestion
FROM golang:1.17.7
```

ref: https://github.com/kubernetes/release/issues/2425